doc/riverctl.1.scd

```diff
@@ -193,10 +193,12 @@ A complete list may be found in _/usr/include/linux/input-event-codes.h_
 *enter-mode* _name_
 	Switch to given mode if it exists.
 
-*map* [_-release_|_-repeat_] _mode_ _modifiers_ _key_ _command_
+*map* [_-to-code_|_-release_|_-repeat_] _mode_ _modifiers_ _key_ _command_
```

Comment:
What do you think about making this flag `-positional` instead? I think that more intuitively explains that this is mapping a position on the physical keyboard rather than the logical keysym received from xkbcommon.

Reply:
I mainly choose `-to-code` because it is already present in sway, `-positional` is also fine for me.

Reply:
@ifreund Another option could be `-independent` for layout-independent. I also like `-static` or `-fixed`.